fetch.bs

```diff
@@ -3348,6 +3396,8 @@ the request.
  <li><p>Run <a>main fetch</a> given <var>fetchParams</var>.
 </ol>
 
+<h3 id="sec-current-time">Current Time</h3>
```

Comment:
I don't think this should be an h3 here, but probably something that you should defined in a different section (e.g. in "infrastructure")

Reply:
You beat me to it :)